libnmstate/metadata.py

```diff
@@ -19,7 +19,14 @@
 
 from libnmstate import iplib
 from libnmstate.appliers import linux_bridge
+from libnmstate.error import NmstateValueError
+from libnmstate.nm import nmclient
```

Comment:
It is more compact and with context to import just nm, then use nm.<module>
I think you just need to include `dns` into nm.__init__